apps/music/index.html

```diff
@@ -43,11 +40,18 @@
     <script defer src="js/ui/title_bar.js"></script>
     <script defer src="js/ui/tab_bar.js"></script>
     <script defer src="js/ui/views/mode_manager.js"></script>
+    <!-- Lazy load code -->
+    <!--
+    <script defer src="shared/js/text_normalizer.js"></script>
+    <script defer src="shared/js/bluetooth_helper.js"></script>
+    <script defer src="shared/js/media/remote_controls.js"></script>
+    <script defer src="js/metadata/*.*"></script>
```

Comment:
Cool. You can assign that bug to me, since I probably know the most about the metadata parser.